<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DruidVolume</class>
 <widget class="QGroupBox" name="DruidVolume">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>394</width>
    <height>163</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GroupBox</string>
  </property>
  <property name="title">
   <string>Volume</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="topMargin">
    <number>40</number>
   </property>
   <item row="0" column="3" rowspan="5">
    <widget class="QLCDNumber" name="_lcd"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Fermentor</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QDoubleSpinBox" name="_height_fermentor"/>
   </item>
   <item row="2" column="2">
    <widget class="QPushButton" name="_compute_fermentor">
     <property name="text">
      <string>Compute</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Boiler</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QDoubleSpinBox" name="_height_boiler"/>
   </item>
   <item row="3" column="2">
    <widget class="QPushButton" name="_compute_boiler">
     <property name="text">
      <string>Compute</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QDoubleSpinBox" name="_height_sterilizer"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Sterilizer</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QPushButton" name="_compute_sterilizer">
     <property name="text">
      <string>Compute</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
